packages/replay/package.json

```diff
@@ -20,7 +20,7 @@
   ],
   "scripts": {
     "prepare": "yarn run build",
-    "build": "rm -rf dist/ metadata/ tsconfig.tsbuildinfo && tsc && node ./scripts/mirror-dist-metadata.js",
+    "build": "pkg-build && node ./scripts/mirror-dist-metadata.js",
```

Comment:
The previous setup forgot to run `node ./scripts/mirror-dist-metadata.js` here 😢 It was the only `build` script that had some extra requirements.

This broke our upload action since it requires files manually instead of relying on node's resolution:
https://github.com/replayio/action-upload/blob/715f00955415fcc6873ce268396489a2ad07dca2/action.yml#L48-L50